docs/utilities/gestures.md

```diff
@@ -139,6 +139,8 @@ const gesture = createGesture({
 
 import Basic from '@site/static/usage/v7/gestures/basic/index.md';
 
+In this example, our app listens for gestures on the `ion-content` element. When a gesture movement is started, the `onStart` function is called and a class is added to our `ion-card` to add a colored box shadow. When a gesture movement is detected, the `onMove` function is called and our app prints the current gesture information within the `ion-card`. Finally, when a gesture movement is ended, the `onEnd` function is called and the custom class is removed from our `ion-card`.
```

Comment:
It feels like `gesture movement is detected` and `gesture movements is ended` come off a bit weird. It might just be me but I wonder if it would be better to switch `is` to `has`. It's a suggestion though.